exchange/client.go

```diff
@@ -12,10 +15,30 @@ func BuildHTTPClient(options *Options) (*http.Client, error) {
 	if options.FollowRedirects {
 		checkRedirect = nil
 	}
+	tlsConf := &tls.Config{}
+
+	if options.SkipVerify {
+		tlsConf.InsecureSkipVerify = true
+	}
+
+	transp := &http.Transport{
```

Comment:
I found that zero value of `http.Transport` is NOT `DefaultTransport`.
We should explicitly `Clone` the `DefaultTransport` and then overwrite `TLSClientConfig`.

This SO answer would help.
https://stackoverflow.com/a/51848441/5552868

In the `DefaultTrasnport`, `http.ProxyFromEnvironment` is already set. So we can remove the line setting it.

Reply:
Done.